index.html

```diff
@@ -24,11 +24,11 @@
 			<h2> Modern Programming for Data Analytics - Class Slides </h2>
 		    </section>
 		    <section>
-			    <h3>Your name</h3>
```

Comment:
You replaced the previous slide and didn't copy leaving the original, please correct.

Reply:
I copied and pasted correctly in the index.html this time but looks like pushed the commit in a wrong way. I Should have created a branch, edited the index.html , but I pulled and edited the main branch and pushed that back.  I don't know how to revert that.  My aplogoies.